extlib/CMakeLists.txt

```diff
@@ -1,18 +1,12 @@
-find_package(HDF5 REQUIRED)
-add_library(HighFive INTERFACE)
-target_include_directories(HighFive INTERFACE
-    HighFive/include/
-    ${HDF5_INCLUDE_DIRS})
-target_link_libraries(HighFive INTERFACE
-    ${HDF5_C_LIBRARIES})
-
+option(HIGHFIVE_USE_BOOST "Disable boost in HighFive" OFF)
+add_subdirectory(HighFive)
+option(FMT_INSTALL "Install fmt for static library" ON)
```

Comment:
do we really want this? are these options exposed on, e.g., to the user via `ccmake`?

Reply:
Yes, users can modify it.

Reply:
I'm not convinced that we should install stuff from submodules. That really smells like dependency conflicts down the line.

Reply:
The question here is: Should we install fmt even if it's a header-only depends?

Reply:
The answer is yes. Otherwise there is no target available for cmake.

Reply:
Errm, why do we need the target? Shouldn't this be a private thing for libSONATA?

Reply:
I'm not following this; why does someone building the project need to have options for our internal dependencies?

Reply:
The option here is use to set the option, not to expose them (they already are exposed)